go/vt/sqlparser/sql.y

```diff
@@ -4778,6 +4778,19 @@ column_list_opt:
     $$ = $2
   }
 
+column_list_empty:
+  {
+    $$ = nil
+  }
+| sql_id
+  {
+    $$ = Columns{$1}
+  }
+| column_list_empty ',' sql_id
+  {
+    $$ = append($$, $3)
+  }
```

Comment:
Why not just use column_list here?

Reply:
Good point, that's nicer indeed. Let me fix that up.

Reply:
@GuptaManan100 Cleaned this up!